src/test/java/org/generationcp/middleware/service/impl/study/StudyEntryServiceImplIntegrationTest.java

```diff
@@ -406,7 +476,18 @@ false, new StudyType(6), "20200606", null, null,
 		this.daoFactory.getProjectPropertyDAO().save(entryNoProp);
 		this.daoFactory.getProjectPropertyDAO().save(crossProp);
 		this.daoFactory.getProjectPropertyDAO().save(entryTypeProp);
+		this.sessionProvder.getSession().refresh(plotDataset);
+		return study;
+	}
+
+	private DmsProject createStudyWithGermplasmListAsociated() {
```

Comment:
rename to createStudyWithGermplasmListAssociated

Reply:
done!